source/Octopus.Tentacle.Client/TentacleClient.cs

```diff
@@ -87,6 +98,19 @@ public async Task<UploadResult> UploadFile(string fileName, string path, DataStr
             return (DataStream?)dataStream;
         }
 
+        internal async Task<CapabilitiesResponseV2> GetCapabilities(ILog logger, CancellationToken cancellationToken)
```

Comment:
This is purely needed to be able to test the caching in an integration test effectively.

Internal so calls clients can't call it directly

Reply:
It would be nice to test this without adding a method that isn't used outside tests. 

Are there any ways we can achieve this?

If not, then that's fine. I thought I'd ask 🙂

Reply:
Could we call the service without going via the client? Or exercise the path when executing the script? I'm not sure how feasible these ideas are though 😞

Reply:
We probably can with decorators!! I'll try and remove the need for this!!

Reply:
I've removed the need for this by refactoring the test to call ExecuteScript and using decorators and a bit of port forwarder orchestration to ensure cached responses are returned.